### PR TITLE
Update Linux Rules

### PR DESCRIPTION
**Pull request by gs3cl**
### Summary of the Pull Request

This PR increases coverage of two linux rules

### Changelog

update: Linux HackTool Execution - Increase coverage by adding more tools
update: Linux Network Service Scanning Tools Execution - Increase coverage by adding more tools

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Review by nasbench — APPROVED** [review]

File: rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    - https://github.com/t3l3machus/hoaxshell
+    - https://github.com/t3l3machus/Villain
+    - https://github.com/HavocFramework/Havoc
+    - https://github.com/1N3/Sn1per
+    - https://github.com/Ne0nd0g/merlin
+author: Nasreddine Bencherchali (Nextron Systems), Georg Lauenstein (sure[secure])
+date: 2023/01/03
+modified: 2023/10/25
+tags:
+    - attack.execution
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml:18]:
```suggestion
    - attack.resource_development
    - attack.execution
    - attack.t1587
```

**Review by frack113 — APPROVED** [review]

File: rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml (hunk 1 above)
Comment (gs3cl) [reply rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml:18]: Thanks !

**Comment by gs3cl** [comment]
@nasbench added default binary from NetExec: https://github.com/Pennyw0rth/NetExec/

**Comment by nasbench** [comment]
> @nasbench added default binary from NetExec: https://github.com/Pennyw0rth/NetExec/
> 

Thanks 👍

File: rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            - '/bpfdos'
+            - '/ebpfki'
+            - '/evil-winrm'
+            - '/hashcat'
+            - '/hoaxshell.py'
+            - '/hydra'
+            - '/john'
+            - '/ncrack'
+        # default binary: https://github.com/Pennyw0rth/NetExec/releases/download/v1.0.0/nxc-ubuntu-latest
+            - '/nxc-ubuntu-latest'
```
Comment (nasbench) [inline rules/linux/process_creation/proc_creation_lnx_susp_hktl_execution.yml:84]:
```suggestion
            # default binary: https://github.com/Pennyw0rth/NetExec/releases/download/v1.0.0/nxc-ubuntu-latest
            - '/nxc-ubuntu-latest'
```

**Review by nasbench — APPROVED** [review]

**Comment by gs3cl** [comment]
@nasbench @frack113 well yesterday I did some testing in my lab. Hack tools rule will be changed later and I will also add example logs. 

I would suggest to wait with merge

Regards,

**Comment by nasbench** [comment]
> @nasbench @frack113 well yesterday I did some testing in my lab. Hack tools rule will be changed later and I will also add example logs. 
> 
> I would suggest to wait with merge
> 
> Regards, 

I suggest otherwise. As this was reviewed already and approved. Please don't add things to it for now :)

Once it's merged you can open a new PR with additional changes.

**Pull request merged**